explainer.md

```diff
@@ -17,86 +15,140 @@ The two drafts apply to both TLS server certificates, where the server authentic
+
+This creates a conflict that clients must navigate. When a client must update its policies to meet new security requirements, it adds to this diversity and the resulting server and CA challenges. The client must then choose between compromising on user security or burdening the rest of the ecosystem. User security usually pays the cost. It can take just one important server with one important other client to interfere with a security transition.
+
+The drafts aim to relieve this conflict, in order to build a robust, secure, and flexible PKI.
+
+## Overview
+
+The two drafts follow the standard negotiation pattern in TLS, extending the existing TLS 1.3 [`certificate_authorities` extension](https://www.rfc-editor.org/rfc/rfc8446#section-4.2.4) to reduce the size cost. They do so with different approaches, with different tradeoffs:
+
+### Trust Expressions
+
+Trust Expressions compresses the certificate authorities list by referencing "trust stores", maintained by root programs. Clients send "trust expressions", which reference these trust stores by name and version, and optionally exclude portions of them. The full CA list is then the union of the trust expressions sent.
+
+When CAs issue certificates to servers, they include "trust store inclusion" metadata, which contain information for servers to evaluate their candidate certificates against the trust expressions.
+
+To support this compression scheme, participating CAs and root programs must coordinate to configure consistent information in clients and servers. To do this, the CA periodically fetches a "trust store manifest" from the root program. The protocol additionally must solve a "version skew" problem where the client references a newer trust store version than the server knows of. Most of the protocol's complexity comes from these two parts.
+
+### Trust Anchor Identifiers
+
+Trust Anchor Identifiers consists of three parts:
+
+1. To mitigate large X.509 names, short "trust anchor identifiers" uniquely identify each participating CA, at an estimated 5 bytes per CA. These are sent in a `trust_anchors` extension, analogous to `certificate_authorities`.
+
+2. Bandwidth and privacy constraints may still prevent a client from enumerating its full CA list. Server use an extension to [HTTPS/SVCB](https://www.rfc-editor.org/rfc/rfc9460.html) to publish available trust anchors in DNS. The client fetches this, selects the desired option, and requests it in `trust_anchors`.
+
+3. To accommodate stale DNS records, or cases where the DNS mechanism does not work, servers also send available trust anchors in the TLS handshake. If the client rejects the certificate, it can use this to retry with a more accurate `trust_anchors` request.
```

Comment:
```suggestion
3. To accommodate stale or missing DNS records, servers also send available trust anchors in the TLS handshake. If the client rejects the certificate, it can use this to retry with a more accurate `trust_anchors` request.
```